MODULE.bazel

```diff
@@ -45,6 +45,12 @@ single_version_override(
     version = "8.16.1",
 )
 
+bazel_dep(name = "rules_jvm_external", version = "6.9")
+single_version_override(
+    module_name = "rules_jvm_external",
+    version = "6.9",
+)
```

Comment:
Please also add a `bazel_dep(name = "rules_jvm_external", version = "6.9")` to `deps/latest/MODULE.bazel`.